azext_edge/edge/_help.py

```diff
@@ -1219,3 +1219,81 @@ def load_iotops_help():
             --sa-resource-id $STORAGE_ACCOUNT_RESOURCE_ID --sa-container myschemacontainer
             -l westus2 --desc 'Contoso factory X1 schemas' --display-name 'Contoso X1' --tags env=prod
     """
+
+    helps[
+        "iot ops connector"
+    ] = """
+        type: group
+        short-summary: Connector management.
+    """
+
+    helps[
+        "iot ops connector opcua"
```

Comment:
will there ever be management for other types of connectors? 

If you are not sure - then keep this

Reply:
yeah there will, I have confirmed with Paymaun before his vacation